.github/workflows/upgrade-python-requirements.yml

```diff
@@ -11,16 +11,44 @@ on:
         default: 'main'
 
 jobs:
-  call-upgrade-python-requirements-workflow:
-    uses: openedx/.github/.github/workflows/upgrade-python-requirements.yml@master
-    with:
-      branch: ${{ github.event.inputs.branch || 'main' }}
-      user_reviewers: edx-revenue-tasks
-      team_reviewers: ""
-      email_address: revenue-tasks@edx.org
-      send_success_notification: false
-    secrets:
-      requirements_bot_github_token: ${{ secrets.REQUIREMENTS_BOT_GITHUB_TOKEN }}
-      requirements_bot_github_email: ${{ secrets.REQUIREMENTS_BOT_GITHUB_EMAIL }}
-      edx_smtp_username: ${{ secrets.EDX_SMTP_USERNAME }}
-      edx_smtp_password: ${{ secrets.EDX_SMTP_PASSWORD }}
+  upgrade_requirements:
+    runs-on: ubuntu-20.04
+
+    strategy:
+      matrix:
+        python-version: ["3.9"]
```

Comment:
Requirements are in python 3.8, is this going to cause an issue?

Reply:
good point. will change. i actually thought we were on 3.9 ><'